server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -53,6 +65,7 @@ class Extender {
     private List<File> extDirs;
     private List<File> manifests;
     private List<File> gradlePackages;
+    private ResolvedPods resolvedPods;
```

Comment:
This class holds a list of pods, the minimum platform version (ios/osx) needed by the resolved pods and the directory where frameworks included in the pods are located